web/libs/datamanager/src/components/Common/Modal/Modal.scss

```
@@ -0,0 +1,128 @@
+.modal {
+  --transition-duration: 100ms;
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 10000;
+  display: flex;
+  position: absolute;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(#000, 0.3);
+  will-change: opacity;
+
+  &__wrapper {
+    width: 100%;
+    max-height: 100%;
+    padding: 40px 0;
+    overflow: auto;
+    box-sizing: border-box;
+    will-change: transform;
+  }
+
+  &__content {
+    width: 400px;
+    min-width: 400px;
+    min-height: 100px;
+    margin: 0 auto;
+    background-color: #fff;
+    border-radius: 5px;
+    box-shadow: 0 10px 30px rgba(#000, 0.3);
+  }
+
+  &__header {
+    display: flex;
+    min-height: 40px;
+    align-items: center;
+    justify-content: space-between;
+    padding: 16px 32px;
+    box-sizing: content-box;
+
+    &_divided {
+      border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+    }
+  }
+
+  &__title {
+    margin: 0;
+    margin-right: auto;
+    font-size: 20px;
+  }
+
+  &__body {
+    padding: 0 40px 32px;
+
+    &_bare {
+      padding: 0;
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  &__footer {
+    padding: 16px 32px;
+    background: rgba(0, 0, 0, 0.03);
+    box-shadow: inset 0px 1px 0px rgba(0, 0, 0, 0.05);
```

Flagging as opportunity for css var replacement. Further review of usage needed.
```suggestion
    background: rgba(var(--black), 0.03);
    box-shadow: inset 0px 1px 0px rgba(var(--black), 0.05);
```